tox.ini

```diff
@@ -64,6 +64,10 @@ source_paths =
     tests/lock_test.py
 
 [testenv]
+passenv = 
+    TRAVIS
+    APPVEYOR
```

Comment:
The first two environment variables should no longer be needed here.

Reply:
Yes you are right. For the CI, either we are on travis and `export TOX_TESTENV_PASSENV=TRAVIS` applies, or we are on appveyor and `set TOX_TESTENV_PASSENV=APPVEYOR` applies.

And outside the CIs, `TRAVIS` and `APPVEYOR` are not set.